docs/guides/admin/docs/configuration/security.user.brightspace.md

```diff
@@ -8,9 +8,10 @@ member of the Brightspace course `myCourseID` will be granted the Opencast role
 
 ### Requirements
 
+<!-- _This section has been left blank! It should be filled with the necessary info, or removed entirely! -->
 
-
-### Step 1
+### Step 1 
+<!-- _Prefer to add titles to steps as in "Step 1: Do this" -->
```

Comment:
I'm fine with whatever as long as the docs are uniform.

Reply:
+1 for titles. They are far more helpful.